nextflow_schema.json

```diff
@@ -58,6 +58,7 @@
                 },
                 "skip_decoy_generation": {
                     "type": "boolean",
+                    "default": false,
```

Comment:
I'm pretty sure if the default is `false` it doesn't need to be included (right @mashehu )?